mycroft/audio/services/mopidy/mopidypost.py

```diff
@@ -86,7 +90,7 @@ def clear_list(self, force=False):
     def add_list(self, uri):
         d = copy(_base_dict)
         d['method'] = 'core.tracklist.add'
-        if type(uri) == str or type(uri) == unicode:
+        if isinstance(uri, basestring):
```

Comment:
We can change this to isinstance(uri, str) since we're not keeping python 2 compatibility.